### ISSUE TITLE
[Support] Rubidium Dynamic Lights

### ISSUE THREAD

**Issue by SeanChengN** [illusivesoulworks/radiantgear#4]
Please support, I think this mod performs better.
https://www.curseforge.com/minecraft/mc-mods/dynamiclights-reforged

**Comment by lynnpye** [comment]
Seconded

**Comment by lynnpye** [comment]
PR submitted: https://github.com/TheIllusiveC4/CuriousLights/pull/5

**Comment by lynnpye** [comment]
Re: PR https://github.com/TheIllusiveC4/CuriousLights/pull/5
I fixed a regression error; my original implementation was literally manipulating the player's light level, but that is what the actual Dynamic Lights Reforged is doing via the PlayerMixin. I altered the implementation to, as with the DynamicLights implementation, create a separate DynamicLightSource to track the CuriousLight. I piggy back on much of the player's implementation (for things like location and update necessity), but made the alterations necessary to track the actual light source's level. I've tested the change both in SMP and via dedicated server/remote client, both with the DynamicLights mod and the DynamicLightsReforged mod, and it seems to be working now.
